NEventSocket/FreeSwitch/EventMessage.cs

```diff
@@ -22,8 +23,20 @@ namespace NEventSocket.FreeSwitch
     [Serializable]
     public class EventMessage : BasicMessage
     {
+        public static readonly byte[] delimiterBytes = Encoding.UTF8.GetBytes("\n\n");
         private static readonly ILogger log = Logger.Get<EventMessage>();
 
+        private static int PatternAt(byte[] source, byte[] pattern)
```

Comment:
This utility function should probably moved a bit more down, constructors are usually the first thing one would expect in a class.